examples/gui_auto.py

```diff
@@ -6,9 +6,10 @@
 
 from wgpu.gui.auto import WgpuCanvas, run
 
-from triangle import setup_drawing_sync
-# from cube import setup_drawing_sync
-
+try:
+    from .triangle import setup_drawing_sync
+except ImportError:
+    from triangle import setup_drawing_sync
```

Comment:
you should likely just move these into a `wgpu/_testing_utils.py` file so you can import them correctly.


Its pretty annoying to not have a "real package structure". 

I honestly don't know a good way to "fix" this without putting these utilities INSIDE the `wgpu` package.

Reply:
I guess the point is to have this stuff in the example itself and not "hidden away" in some packaged file to make it feel like you have a very high level API...

Reply:
I guess this is the warning:

```
examples/tests/test_examples.py::test_examples_run[triangle]
  /home/mark/miniforge3/envs/dev/lib/python3.10/runpy.py:126: RuntimeWarning: 'examples.triangle' found in sys.modules after import of package 'examples', but prior to execution of 'examples.triangle'; this may result in unpredictable behaviour
    warn(RuntimeWarning(msg))
```